docs/_data/components/schemas/headers.yaml

```diff
@@ -10,13 +10,16 @@ proposal-header: &header
   properties:
     source:
       $ref: ./data-provenance.yaml
+    proposal-begins:
```

Comment:
@getorymckeag This is an important change. The overall PR isn't done (still need to bang out an article on the supported media types), but I wanted to get your 👀 on this.